woocommerce/woocommerce
# Update d3-scale to patch vulnerabilities

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Update d3-scale to patch vulnerabilities







### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Open the application and perform smoke tests in the Analytics reports, since it's where the Chart component is used, which relies on the d3-scale library. Pay attention to the charts functionality and compare it to the trunk version.
2. Run `pnpm install` and `pnpm audit` and verify that there are no issues related to d3-color.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
